superset/charts/commands/export.py

```diff
@@ -43,7 +43,7 @@ class ExportChartsCommand(ExportModelsCommand):
     not_found = ChartNotFoundError
 
     @staticmethod
-    def _export(model: Slice) -> Iterator[Tuple[str, str]]:
+    def _export(model: Slice, export_related: bool = True) -> Iterator[Tuple[str, str]]:
```

Comment:
nit: i like `export_dependencies` instead of related

Reply:
I used `related` to keep the same terminology used by FAB, and also because some of these are not technically dependencies (eg, when we export a database you also get the datasets, though they're not dependencies).